sensor_msgs/PointCloud2 organized_scan
---
mesh_msgs/TriangleMeshStamped organized_fast_mesh